example-get-started/code/.github/workflows/cml.yaml

```diff
@@ -20,20 +20,24 @@ jobs:
             PREVIOUS_REF=main
           fi
 
-          echo "# CML Report" > report.md
-          echo "## Plots" >> report.md
+          cat <<EOF > report.md
+          # CML Report
+          ## Plots
+          ![Precision & Recall](./prc.svg)
+          ![Confusion Matrix](./confusion.svg)
+
+          ## Metrics and Params
+          ### ${PREVIOUS_REF} → workspace
+          EOF
+
           dvc plots diff $PREVIOUS_REF workspace \
             --show-vega --targets evaluation/plots/precision_recall.json > vega.json
           vl2svg vega.json prc.svg
-          cml publish prc.svg --title "Precision & Recall" --md >> report.md
 
           dvc plots diff $PREVIOUS_REF workspace \
             --show-vega --targets evaluation/plots/confusion_matrix.json > vega.json
           vl2svg vega.json confusion.svg
-          cml publish confusion.svg --title "Confusion Matrix" --md >> report.md
 
-          echo "## Metrics and Params" >> report.md
-          echo "### $PREVIOUS_REF → workspace" >> report.md
           dvc exp diff $PREVIOUS_REF --show-md >> report.md
```

Comment:
```suggestion
          cat <<EOF > report.md
          # CML Report
          ## Plots
          ![Precision & Recall](./prc.svg)
          ![Confusion Matrix](./confusion.svg)

          ## Metrics and Params
          ### ${PREVIOUS_REF} → workspace
          $(dvc exp diff $PREVIOUS_REF --show-md)
          EOF
```

Reply:
executing a command in the middle of a heredoc gives me the shivers

Reply:
Submitted an alternative - ptal